Library/Formula/ne.rb

```diff
@@ -0,0 +1,17 @@
+class Ne < Formula
+  desc "ne, the nice editor"
```

Comment:
Please don't start formula descriptions with the formula name. That happens automatically.